activejob/lib/active_job/queue_adapters.rb

```diff
@@ -49,12 +49,14 @@ module QueueAdapters
     autoload :SuckerPunchAdapter
     autoload :TestAdapter
 
-    ADAPTER = 'Adapter'.freeze
-
     class << self
       def lookup(name)
         const_get(name.to_s.camelize << ADAPTER)
       end
     end
+
+    private
+
+    ADAPTER = 'Adapter'.freeze
```

Comment:
This unfortunately does not make the constant private:

```ruby
>> class Foo
>>   private
>>   ADAPTER = 'Adapter'.freeze
>>   end
=> "Adapter"
>> Foo::ADAPTER
=> "Adapter"
```

What you need is to use `private_constant` instead:

```ruby
>> Foo.private_constant :ADAPTER
=> Foo
>> Foo::ADAPTER
NameError: private constant Foo::ADAPTER referenced
```